opencensus/metrics/export/time_series.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.metrics.export.metric_descriptor import MetricDescriptorType
+
+
+class TimeSeries(object):
+    """Time series data for a given metric and time interval.
+
+    This class implements the spec for v1 TimeSeries structs as of
+    opencensus-proto release v0.0.2. See opencensus-proto for details:
+
+        https://github.com/census-instrumentation/opencensus-proto/blob/24333298e36590ea0716598caacc8959fc393c48/src/opencensus/proto/metrics/v1/metrics.proto#L112  # noqa
+
+    A TimeSeries is a collection of data points that describes the time-varying
+    values of a metric.
+
+    :type start_timestamp: int
```

Comment:
type ```time```?

Reply:
What's a `time`? Changed to `str` to match the existing tests for other ts fields, but we should consider using datetimes (or epoch ms/ns) everywhere instead.